README.md

```diff
@@ -1,7 +1,7 @@
 linux-system-roles/network
 ==========================
 [![Coverage Status](https://coveralls.io/repos/github/linux-system-roles/network/badge.svg)](https://coveralls.io/github/linux-system-roles/network)
-[![Travis Build Status](https://travis-ci.com/linux-system-roles/network.svg?branch=master)](https://travis-ci.com/linux-system-roles/network)
+[![Travis Build Status](https://travis-ci.com/linux-system-roles/network.svg?branch=main)](https://travis-ci.com/linux-system-roles/network)
```

Comment:
good catch. It does not belong in this PR, strictly speaking.

Reply:
Moved to another PR. Thanks!